plugins/inputs/pgbouncer/pgbouncer.go

```diff
@@ -0,0 +1,196 @@
+package pgbouncer
+
+import (
+	"bytes"
+	"database/sql"
+	"regexp"
+	"strings"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/inputs"
+
+	"github.com/lib/pq"
```

Comment:
it was just a suggestion.

I'm actually curious why a separate plugin is necessary for pgbouncer. I think if the version check was made optional in the postgresql_extensible it'd work for pgbouncer as well.